src/dan/Dan.jsx

```diff
@@ -1,38 +1,168 @@
+import React from 'react'
+import myImage from '../assets/e948592c6c5a1022f18fe18d527af199.png'
 
-import "./Dan.css";
-import Navbar from "./components/Navbar";
-import About from "./components/About";
-import Projects from "./components/Projects";
-import Home from "./components/Home";
-import 'bootstrap/dist/css/bootstrap.min.css';
-import Contact from "./components/Contact";
-import Experience from "./components/Experience";
+import './Dan.css'
 
-function Dan() {
-  return (
-    <div className="App">
-      <Navbar />
-      <div id="home">
-        <Home />
-      </div>
-
-      <div id="about">
-        <About />
-      </div>
+const Dan = () => {
+  const encodedAddress = encodeURIComponent(
+    '123 N State St., Chicago, IL, 60656'
+  )
 
-      <div id="experience">
-        <Experience />
-      </div>
-
-      <div id="projects">
-        <Projects />
+  return (
+    <div className="main-content">
+      <div className="header-image">
+        <div>
+          <img src={myImage} alt="logo" width={'250px'} />
+        </div>
+        <div className="text-headerImg">
+          <p className="headerText">Header image </p>
+        </div>
       </div>
-
-      <div id="contact">
-        <Contact />
+      <div className="content-box">
+        <div className="alignIcons">
+          <div className="text-header">
+            <h1>ITF Logistics</h1>
+            <p>www.itflogistics.com</p>
+          </div>
+          <div className="social-icons">
```

Comment:
@dan-begaliev  Not all companies may have all social links, so figure it out to make it dynamic